clustered/integration-test/build.gradle

```diff
@@ -16,11 +16,16 @@
 
 import org.gradle.internal.jvm.Jvm
 
+configurations {
+  serverClasspath
+}
+
 dependencies {
   testCompile project(':dist')
   testCompile project(':clustered:clustered-dist')
   testCompile project(':management')
   testCompile "org.terracotta.management:management-entity-client:$parent.managementVersion"
+  testCompile "org.terracotta.management:monitoring-service-entity:$parent.managementVersion"
```

Comment:
No problem for me - was just curious and now I understand why I failed when trying to do the same